.github/workflows/GPU.yml

```diff
@@ -0,0 +1,41 @@
+name: GPU
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: [self-hosted, GPU, benchmark]
+
+    steps:
+      - name: cleanup
+        run: rm -rf *
+      - name: Checkout votca repo
+        uses: actions/checkout@v2
+        with:
+          repository: votca/votca
+          submodules: true
+          ref: master
```

Comment:
When we ultimately want enable it for stable, that should be the target branch, so something like `${{ github.base_ref }}` for pull request and `${{ github.ref }} for branch builds.

Reply:
how can I do either?

Reply:
See my comment below.